css/css-contain/contain-layout-001.html

```diff
@@ -11,9 +11,14 @@
 <style>
 span {
   contain: layout;
+  display: inline-block;
```

Comment:
Delete the `display: inline-block;` line. we're trying to test the absence of effect on non-atomic inlines, and this makes it into an atomic inline. Maybe an accidental leftover from debugging?

Reply:
Ooops, yeah sorry I was checking some things there and I forgot to delete it later.